README.md

```diff
@@ -47,7 +47,7 @@ Our site makes use of certain environment variables. Defaults are provided, but
 | ENV VAR           | DEFAULT VALUE (if not set)       | PURPOSE 
 |-------------------|----------------------------------|-----------------------------------------------------|
 | `ENV`             | prod                             | Determine bundling optimizations and feature flags  | 
-| `USASPENDING_API` | https://api.usaspending.gov/api/ | API URL to get backend data                         |
+| `USASPENDING_API` | https://api.usaspending.gov      | API URL to get backend data                         |
```

Comment:
This value was kind of right before, depends on the goal of this. When hitting the API all endpoints are going to start with `https://api.usaspending.gov/api/`. But the actual URL for the API is `https://api.usaspending.gov`